### PR TITLE
Update docs to use maximum number of 2^256 items

### PR DESCRIPTION
**Pull request by pix16**
type uint256 has 256 bits, which should allow for 2^256 items (from 0 to (2^256)-1). Similar to how uint8 should allow for 2^8 (256) items (from 0 to 255 aka (2^8) - 1).

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by lightclient** [comment]
Waiting for approval from @pizzarob.

**Review by pizzarob — APPROVED** [review]
LGTM

**Review by MicahZoltu — APPROVED** [review]
Testing to see if editor approval will override Final.

**Pull request merged**
